sources/notion/index.js

```diff
@@ -62,7 +62,13 @@ const _getJournals = async ({ database_id, filters: { start, end }, isDaily }) =
         name: _Name.title[0].plain_text.split(' ')[0],
         LWD: _lwd ? _lwd.map(({ plain_text, href }) => {
           if (href) {
-            return `[${plain_text}](${href})`
+            // match * symbol followed by whitespace
+            // (format by split('* ') below cause incorrect url link in post)
+            const match = plain_text.match(/(^\*) (?<body>.*)/)
+            let text = plain_text
+
+            if (match) text = match.groups.body
```

Comment:
if it's a solo if with assignment involved, prefer ternary

otherwise having a curly-braces block is more indicative that this is a conditional flow

Reply:
fixed!